File: 1295-Find-Numbers-with-Even-Number-of-Digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
const findNumbers = function(nums) {
  let amount = 0;
  let x = 0;
  
  for (let n of nums) {
    x = 0;
    while(Math.trunc(n)) {
      x++;
      n /= 10;
    }
    // console.log(x)
    if (x%2 == 0) amount++;
  }
  
  return amount;
};

File: solutions/1295-Find-Numbers-with-Even-Number-of-Digits.cpp
class Solution {
public:
  int findNumbers(vector<int>& nums) {
    int count {0};
    for (auto it = nums.begin(); it != nums.end(); ++it) {
      auto s = std::to_string(*it);
      if (s.size() % 2 == 0) count++;
    }
    
    return count;
  }
};
